prism/src/main/java/org/apache/falcon/service/BacklogMetricEmitterService.java

```diff
@@ -126,17 +132,21 @@ public void onChange(Entity oldEntity, Entity newEntity) throws FalconException{
                 backlogMetricStore.deleteEntityInstance(newProcess.getName());
                 entityBacklogs.remove(newProcess);
             }
+            Process process = EntityUtil.getEntity(oldEntity.getEntityType(), oldEntity.getName());
```

Comment:
@pallavi-rao  We have made the changes but one question is coming to mind after this change, if we stop populating this data how would user differentiate if the falcon service is down or its because process has been deleted?